ci-project/models/overview.md

```diff
@@ -9,3 +9,21 @@ This [dbt](https://www.getdbt.com/) project is for demonstrations and tutorials.
 The source code can be found [here](https://github.com/clrcrl/jaffle_shop).
 
 {% enddocs %}
+
+{% docs __jaffle_shop__ %}
+## Jaffle_shop Project level overview example
+This [dbt](https://www.getdbt.com/) project may have only one project (`jaffle_shop`) currently.
+However there may be other projects included in your own projects.
+
+You can assign a unique overview for each project by adding a docs block in an .md file of your project
+<<<<<<< HEAD
+and giving it a name with the following convention { \__project_name__ }
+{% enddocs %}
+
+{% docs __dbt_utils__ %}
+## DBT_UTILS package overview
+This [dbt package](https://docs.getdbt.com/docs/building-a-dbt-project/package-management) is a collection of tools to help with common tasks.
+=======
+and giving it a name with the following convention __<project_name>__.
+>>>>>>> parent of b2999b6... Added overview block for dbt_utils, and generated new docs from ci_project(replaced existing in data/)
```

Comment:
Tiny thing: looks like there are a few git vestiges from merging

Reply:
Ah my bad, next commit should fix.